common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalBlockHandler.java

```diff
@@ -89,6 +110,21 @@ public void receive(TransportClient client, ByteBuffer message, RpcResponseCallb
     handleMessage(msgObj, client, callback);
   }
 
+  @Override
+  public StreamCallbackWithID receiveStream(
```

Comment:
rename to `receivePushBlockStream` ?

Reply:
This is from the API introduced in SPARK-6237.
That API was used for transferring large RDD partition blocks (>2GB), and we reused it for transferring shuffle partition blocks.
Prefer to keep this API's name generic in case there are other use cases that could reuse this as well.